src/E1_process_results.py

```diff
@@ -722,3 +727,72 @@ def convert_costs_to_dataframe(dict_values):
     df_pie_plot.iloc[-1, 0] = "Total"
 
     return df_pie_plot
+
+def convert_scalars_to_dataframe(dict_values):
```

Comment:
It seems to me that you could build `kpi_scalars_dataframe` simply with `pd.Dataframe.from_dict({KPI: kpi_names, UNIT: units_cost_kpi})` if you can get the `kpi_names` directly from `dict_values[KPI][KPI_SCALARS_DICT]`

Reply:
Hmhm, I actually do not now how to do it better right now. I would leave it for as I expect more changes to this function and also to the function generating tables in the report.